PoolArn attribute support in Data Source: aws_ec2_coip_pool (#17046)
hashicorp/terraform-provider-aws: website/docs/d/ec2_coip_pool.html.markdown

> The following attribute is additionally exported:
> 
> * `pool_cidrs` - Set of CIDR blocks in pool
> 
> * `pool_arn` - ARN of the COIP pool

Reviewer: ```suggestion
In addition, the following attributes are exported:

* `arn` - ARN of the COIP pool
* `pool_cidrs` - Set of CIDR blocks in pool
```
